### ISSUE TITLE
Assigning selected image from photo library to View2 (a view controller) does not work

### ISSUE THREAD

**Issue by dyHub** [dyHub/BirdLookUpV1#2]
Function that was being called when user chooses an image from the photo library :

```
func imagePickerController(picker: UIImagePickerController,  didFinishPickingMediaWithInfo info: [String : AnyObject]) {
        // The info dictionary contains multiple representations of the image, and this uses the original.
        let selectedImage = info[UIImagePickerControllerOriginalImage] as! UIImage

        // Dismiss the picker.
        dismissViewControllerAnimated(true, completion: nil)

        // connect the code with the view2 story board
        var View2VC = self.storyboard?.instantiateViewControllerWithIdentifier("view2") as!View2

        // Set view 2 image View to display the selected image.
        View2VC.cameraImage.image = selectedImage

        let navigationController = UINavigationController(rootViewController: View2VC)

        // show view2 view controller
        presentViewController(navigationController, animated: true, completion: nil)

    }
```

Code that was supposed to process when assign selected image to the view controller which would determine the subject of the image:

```
View2VC.cameraImage.image = selectedImage
```

This line is not working
Debugging:

```
 Thread1: EXC_BAD_INSTRUCTION (code=EXC_1386_INVOP, subcode=0x0)
```

**Comment by itstehkman** [comment]
The issue was that running this line of code: 

```
var View2VC = self.storyboard?.instantiateViewControllerWithIdentifier("view2") as!View2
View2VC.cameraImage.image = selectedImage
```

does not instantiate any IBoutlets until after viewDidLoad is called. This only initializes the view, but viewDidLoad is never called, so the IBoutlets are still nil. CameraImage (an IBoutlet) is nil, so you cannot access any properties of a nil object, and thus won't change anything.

**Comment by itstehkman** [comment]
I fixed this in commit 20087e7, check it out.
